lib/signaling/v2/peerconnection.js

```diff
@@ -605,7 +605,8 @@ class PeerConnectionV2 extends StateMachine {
       }));
       return transceiver;
     }
-    return this._peerConnection.addTransceiver(track);
+    // TODO(lrivas): Review with Charlie, the second argument is required by VDI environment
+    return this._peerConnection.addTransceiver(track, {});
```

Comment:
This is fine. You can update the comment to
> // Pass an empty object for init options. Citrix fails on undefined or null.

Reply:
I decided to revert the change and fill in the gap with the polyfills. I feel it’s better to stick to the standard approach for adding a transceiver. Thanks for the feedback!